### DISCUSSION TITLE
MaxMind integration not reflected in Ui

### DISCUSSION THREAD

**Discussion by tomtom215** [plausible/analytics#1657] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug

I have followed the documentation for MaxMind integration. I have used both a legacy and new MaxMind API key format and run the combined docker-compose command to start the geoIP container.

Everything starts fine, I can see the logs of the geoIP container that the MaxMind DB has been pulled. Nothing is reflected in the Ui to show more granular location data. I tried visiting my monitored website after applying these changes but I still only can select "Countries" in the Ui.

### Expected behavior

To add my MaxMin account info the to geoIP config file, run the combined docker-compose up command and have Plausible now offer city granularity insight like the hosted Plausible instances do

### Environment

```markdown
- OS:Docker 
- Browser: Chrome
- Browser Version: Latest
```

**Comment by julianwachholz** [comment, marked as answer]
https://github.com/plausible/analytics/discussions/1473
